src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,85 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+
+public class Calculator {
+    int a = Integer.MIN_VALUE;
+    int b = Integer.MIN_VALUE;
+
+    String operand = "";
+
+    public int add(int a, int b) {
+        return a+b;
+    }
+    public int substract(int a, int b) {
+        return a-b;
+    }
+    public int multiply(int a, int b) {
+        return a*b;
+    }
+    public int divide(int a, int b) {
+        return a/b;
+    }
```

Comment:
스페이스, 개행 컨벤션을 확인해 보면 좋을 것 같습니다. 

```suggestion
    public int add(int a, int b) {
        return a + b;
    }
    
    public int substract(int a, int b) {
        return a - b;
    }
    
    public int multiply(int a, int b) {
        return a * b;
    }
    
    public int divide(int a, int b) {
        return a / b;
    }
```